### ISSUE TITLE
Mumble drops silently after a while

### ISSUE THREAD

**Issue by ghost** [mumble-voip/mumble#1289]
I been having a very strange issue with Mumble since I first started using it (which has been about a year ago): Mumble drops out of voice randomly without any notification. The GUI remains completely the same and it appears as if you're still in the server, even the voice activity button next to your name will light up when you speak, but you will not be able to hear anyone else (their buttons won't light up either) nor will they be able to hear you. In short you're "disconnected" from the server without actually being disconnected.

What I can give as information about this is that I've tried many different solutions: I'm currently on Linux, but the same issue occurs in Windows. It also occurs on different hardware. I've tried fiddling with pretty much every option (from QoS to TCP to audio per packet to using IPv6 instead of IPv4). It occurs on my own server as well as public servers. The only thing that springs to mind that could be the cause is my internet connection. I have very small drops from time to time (so small that usually they're too small for me to pick them up while browsing). Mumble sometimes seems to pick these up and correctly disconnects and then reconnects. The same applies when I quickly replug the cable. What's very strange is that a few friends of mine have the same provider and subscription and are even located closely to my location and do not experience this issue. My brother, who lives in the same house, has exactly the same issue with Mumble. We have no such issues with other voice software (we used to use Skype before Mumble).

Could it perhaps be that the drops are so small that Mumble does not even pick them up, happily continuing in some kind of disconnected state where it still thinks it's connected? My guess is also that it's a client issue and not a server issue (we tried murmur as well as umurmur for our server).

**Comment by InfernoZeus** [comment]
I've just come across the the same issue. I'm also on Linux.

**Comment by Krzmbrzl** [comment]
Does anyone know, if this bug persists in 1.3?

**Comment by Gert-dev** [comment]
@Krzmbrzl Thanks for following up.

I also experienced this issue in the past. The issue "magically" solved itself when I got a new router; my old one was, as indicated in the OP, causing very minor packet drops, small enough to not be noticed by most applications (or they were quickly recovered from), but apparently large enough to generate problems for Mumble. Occasionally I also noticed this in other applications.

After that, a friend of mine moved to another place and suddenly started experiencing the same drops (had the same router model as I did back then). He hasn't changed anything since then, but the issue hasn't appeared since quite some time ago, which could very well be since the release of 1.3, so this can be seen as fixed from my end, at least.
